addons/payment/views/payment_templates.xml

```diff
@@ -56,14 +56,72 @@
             <t t-set="footer_template_id"
                t-value="footer_template_id or 'payment.footer'"/>
 
+            <!-- === Tokens === -->
+            <t t-if="token_count>0"> 
+                Saved payment methods
+                <div class="card">
+                    <t t-foreach="tokens" t-as="token">
+                        <div name="o_payment_option_card" class="card-body o_payment_option_card">
+                            <label>
+                                <!-- === Radio button === -->
+                                <input name="o_payment_radio"
+                                    type="radio"
+                                    t-att-checked="token.id == default_payment_option_id"
+                                    t-att-data-payment-option-id="token.id"
+                                    t-att-data-provider="token.provider_code"
+                                    data-payment-option-type="token"/>
```

Comment:
Could you please realign all the attributes as that as the case before on the whole diff? ^^